java/pom.xml

```diff
@@ -29,6 +30,12 @@
             <version>${lombok.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.amazon.deequ</groupId>
+            <artifactId>deequ</artifactId>
+            <version>${deequ.version}</version>
```

Comment:
We need to change this. Otherwise we won't get the changes we made.

Reply:
this requires building deequ from our branch before we build hsfs. not sure what needs to be done for this.

Reply:
I think there is already a pipeline to build deequ. I can take care of fixing the group name.

Reply:
I published the our fork of deequ using the `com.logicalclocks` group, so that is easier to manage: https://archiva.hops.works/#artifact/com.logicalclocks/deequ